github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -683,7 +672,7 @@ presubmits:
       preset-podman-in-container-enabled: "true"
       vgpu: "true"
     max_concurrency: 1
-    name: pull-kubevirt-e2e-kind-1.23-vgpu-nonroot
+    name: pull-kubevirt-e2e-kind-1.23-vgpu-root
```

Comment:
I think we need to make `pull-kubevirt-e2e-kind-1.23-vgpu` required again so that we are testing the non-root case. This could probably be moved to optional too as we just want one required lane of sriov and vgpu.

Reply:
done